docs/cimpl6ToolingReference.md

```diff
@@ -342,83 +343,69 @@ The Content Profile (CP) specifies the profiles to be included in the IG. As exp
 
-MustSupport elements are not part of the data model, because different elements are important for different use cases. For example, one use case may require a patient's date of death and another might not, but the data model for date of death should be the same wherever it is used.
+#### Content Profile Example
 
-#### Specifying No-Profile Elements
+Here is an example of a CP file:
 
-The No-Profile (`NP`) tag instructs SHR-CLI to NOT profile a class, and therefore, exclude it from the list of profiles.
+```
+    Grammar: ContentProfile 1.0
 
-> In clinical models, classes frequently reference other classes. A laboratory result may reference a patient, a specimen, a device, etc. In turn, the specimen could reference a procedure, the procedure its location, the device its manufacturer, etc. There could be large network of directly- and indirectly-required classes that extend well beyond the intended scope of the IG. If any of these classes have differentials with respect to the resources they are mapped to, then by default, SHR-CLI will create profiles for them, and bring those profiles into the IG. The `NP` flag is a way to put boundaries around your IG.
+    Namespace: obf.datatype NP
 
-Because dependencies can be hard to predict, a good way to determine which classes should be designated with an `NP` flag is to generate the IG, then go to the `out/fhir/profiles` directory to see the complete list of profiles generated. If any profiles in that directory are extraneous to the purpose of the IG, go back to the CP and apply the `NP` flag to the corresponding class.
+    Namespace: obf
+        Patient:
+            BirthDate MS
+        Encounter: NP
+        EpisodeOfCare: NP
 
-The `NP` flag should not be applied to any class that is a parent class of one of the classes you want in your IG. This may interfere with mapping rules inherited from that parent class.
+    Namespace: vital *
+        BloodPressure:
+            Components.ObservationComponent MS
+            RelevantTime MS
+        BodyHeight:
+            DataValue MS
+            RelevantTime MS
+        BodyWeight:
+            DataValue MS
+            RelevantTime MS
+        HeadCircumference: NP
 
-***
-**NOTE:** The "No Profile" feature is available in SHR-CLI 6.6.0 and higher
-***
+```
+In this example, no classes from `obf.datatype` are to be profiled. In the `obf` namespace, only `Patient` is to be included in the IG. `Patient.BirthDate` is to be marked **MustSupport**. `Encounter` and `EpisodeOfCare` are not be profiled. All `Entries` in the namespace `vital` are to be be included, except for `HeadCircumference`. `BloodPressure`, `BodyHeight`, and `BodyWeight` have `MustSupport` elements specified.
```

Comment:
`Encounter` and `EpisodeOfCare` are not **to be** profiled.  Last sentence for `vital` namespace is confusing between what `Entries` are included and what ones have **MustSupport** **elements**

Reply:
ok